Tests/Common/SonarQube/TestLogger.cs

```diff
@@ -210,7 +210,12 @@ private static void WriteLine(string message, params object[] args)
 
         private static string GetFormattedMessage(string message, params object[] args)
         {
-            return string.Format(System.Globalization.CultureInfo.CurrentCulture, message, args);
+            string formatted = message;
+            if (args.Any())
+            {
+                formatted = string.Format(System.Globalization.CultureInfo.CurrentCulture, message, args);
+            }
+            return formatted;
```

Comment:
Previously errored trying to process messages with no arguments that contained empty java classes e.g. "class MyClass{}".